src/settings.rs

```diff
@@ -29,10 +29,22 @@ pub(crate) fn metering_print() -> u64 {
     METERING.print()
 }
 
+pub(crate) fn metering_create_sc_ratio() -> usize {
+    METERING.create_sc_ratio()
+}
+
 pub(crate) fn metering_remaining_points() -> u64 {
     METERING.remaining_points()
 }
 
+pub(crate) fn metering_set_data_ratio() -> usize {
```

Comment:
maybe metering_set_data_gas_per_byte and then just multiply data.len() with it, so that we don't have to do divisions, ratios, roundings etc...

Reply:
I've done something like that in the next commit. We can maybe refactor names later!